pyCGM_Single/pycgmStatic.py

```diff
@@ -215,49 +435,78 @@ def headoffCalc(axisP, axisD):
+
     Calculates the offset angle between that two axis.
 
     It is rotated from uncorrect axis in YXZ order.
     
-    ----------------------------------------------------------------
-    INPUT: Two arrays of uncorrect axis and correct axis. 
-            corect axis depends on foot flat options.
+    Parameters
+    ----------
+    frame : dict 
+        Dictionaries of marker lists.
+            { [], [], [], ... }
+    ankle_JC : array
+        An array containing the x,y,z axes marker positions of the ankle joint center. 
+    knee_JC : array
+        An array containing the x,y,z axes marker positions of the knee joint center. 
+    flat_foot : boolean
+        A boolean indicating if the feet are flat or not.
+    vsk : dict, optional
+        A dictionary containing subject measurements from a VSK file.
+            { [], [], [], ... }
             
-    OUTPUT: Returns offset angle.
+    Returns
+    ------- 
+    angle : array
+        An offset angle.
             [[Right Flexion angle, Right Abduction angle, Right Rotation angle],
             [Left Flexion angle, Left Abduction angle, Left Rotation angle]]
      
-    MODIFIES: the correct axis changes following to the foot flat option. 
-    ----------------------------------------------------------------
-    
-    EXAMPLE:
-            i=1
-            knee_JC: [array([364.17774614, 292.17051722, 515.19181496]),
-                    array([143.55478579, 279.90370346, 524.78408753]),
-                    array([[[364.64959153, 293.06758353, 515.18513093],
-                            [363.29019771, 292.60656648, 515.04309095],
-                            [364.04724541, 292.24216264, 516.18067112]],
-                           [[143.65611282, 280.88685896, 524.63197541],
-                            [142.56434499, 280.01777943, 524.86163553],
-                            [143.64837987, 280.04650381, 525.76940383]]])]
-            ankle_JC: [array([393.76181608, 247.67829633, 87.73775041]),
-                        array([98.74901939, 219.46930221, 80.6306816]),
-                        [[array([394.4817575, 248.37201348, 87.715368]),
-                        array([393.07114384, 248.39110006, 87.61575574]),
-                        array([393.69314056, 247.78157916, 88.73002876])],
-                        [array([98.47494966, 220.42553803, 80.52821783]),
-                        array([97.79246671, 219.20927275, 80.76255901]),
-                        array([98.84848169, 219.60345781, 81.61663775])]]]
-                delta: 0 
-                
-            staticCalculation(frame,ankle_JC,knee_JC,flat_foot,vsk=None)
-            
-            >>> [[0.03482194, 0.14879424, 0],
-                [0.01139704, 0.02142806, 0]]
+    Modifies
+    --------
+    The correct axis changes following to the foot flat option. 
+    
+    Example
+    -------
+    >>> import numpy as np
+    >>> from math import *
+    >>> from .pycgmStatic import *
```

Comment:
It would be safer for these examples to import only the functions needed. So instead of *, `from .pyCGMStatic import staticCalculation`

Reply:
Resolved, added import statements for all functions used in a doctest.